ext/storage_sources/azure_store/azure_connection.cpp

```diff
@@ -31,16 +31,38 @@
 #include <azure/core.hpp>
 #include <azure/storage/blobs.hpp>
 
+#include <stdlib.h>
+#include <iostream>
+#include <vector>
+#include <string>
+
 azure_connection::azure_connection(const std::string &bucket_name, const std::string &obj_prefix)
     : _azure_client(Azure::Storage::Blobs::BlobContainerClient::CreateFromConnectionString(
         std::getenv("AZURE_STORAGE_CONNECTION_STRING"), bucket_name)),
       _bucket_name(bucket_name), _object_prefix(obj_prefix)
 {
 }
 
+/*
+ * Lists all of the objects (blobs) on the bucket (container) utilising the listBlobs function form
```

Comment:
Love the commenting, I would reword this more of: "Build a list of all of the objects from the bucket." We should describe the concept of blobs and container either the constructor or the .h file.

Reply:
ok I've added a comment before the contructor:
Buckets are synonymous with Containers, and Objects are synonymous with Blobs.